common/modifiers/babylon_modifiers.txt

```diff
@@ -57,6 +57,11 @@ player_construction_bonus = {
 	country_construction_add = 1
 }
 
+bureaucracy_bonus = {
```

Comment:
game_start_bureaucracy_bonus, you can be really clear with multiplier names for very specific purposes.